docs/reference/commandline/dockerd.md

```diff
@@ -1334,6 +1335,19 @@ For example, the daemon fails to start if you set daemon labels
 in the configuration file and also set daemon labels via the `--label` flag.
 Options that are not present in the file are ignored when the daemon starts.
 
+The `--validate` option allows to validate a configuration file without
+starting the Docker daemon.
+
+```console
+$ dockerd --validate --config-file=/tmp/valid-config.json
+Config OK
```

Comment:
This changed to 

```suggestion
configuration OK
```

Reply:
Should we also mention that the exit status will be 0 (success) and non-zero (failure)? as that's the "actual" check (message that's printed is informational)?